examples/one-page.html

```diff
@@ -22,7 +22,7 @@
     <script src="https://unpkg.com/react-is@16.10.1/umd/react-is.production.min.js"></script>
     <script src="https://unpkg.com/prop-types@15.7.2/prop-types.min.js"></script>
     <!-- <script src="https://unpkg.com/spectacle/dist/spectacle.js"></script> -->
-    <script src="../dist/spectacle.js"></script>
+    <script src="../dist/umd/spectacle.js"></script>
```

Comment:
This isn't working..

**Expected**: A full export of a `Spectacle` object that then hooks up our one-page inline JS/htm content.

**Actual**: `dist/umd/spectacle.js` appears to be built as an _application_ doing the `ROOT/index.js` demo (with MDX by default) that runs and shows a presentation. As soon as we get to our inline script tage, it errors out with `Uncaught ReferenceError: Spectacle is not defined` because presumably it's not an actual UMD exported lib.

Reply:
My bad, needed to change the source location to `components/index.js` rather than `~/index.js`.
I'm inclined to keep the `package.json` `source` field as `components/index.js` to reflect this as the 'library entry' rather than the dev-server entry, what's your opinion on this?

Reply:
`src/components/index.js` doesn't even exist anymore. We've removed it and moved **everything** to https://github.com/FormidableLabs/spectacle/blob/task/rewrite/src/index.js as we're shipping more than components in our final deliverable bundle.